### PR TITLE
ACS-6498: use api instead of sns

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

**Review by tpage-alfresco — APPROVED** [review]
This is great, but the small footprint of the change does highlight our need for some E2E tests.

**Comment by Pawel-608** [comment]
yep I was really suprised pipeline passed😂

File: live-ingester/pom.xml [hunk 1]
```diff
@@ -32,16 +36,18 @@
 			<artifactId>camel-activemq-starter</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.apache.camel.springboot</groupId>
-			<artifactId>camel-sjms-starter</artifactId>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-http</artifactId>
```
Comment (krdabrowski) [inline live-ingester/pom.xml:40]: You can use here:
`<dependency>
	<groupId>org.apache.camel.springboot</groupId>
	<artifactId>camel-http-starter</artifactId>
</dependency>`
which already comes with `httpclient` inside

Comment (Pawel-608) [reply live-ingester/pom.xml:40]: I think it is not camel that needs it, but `SignedStorageClient` and it previously was supplied as some transitive dependency - so it is better to make direct dependency rather that rely on another transitive one

Comment (krdabrowski) [reply live-ingester/pom.xml:40]: Ok, `SignedStorageClient` won't need it any more, but I can remove it in my PR. LGTM

**Review by krdabrowski — APPROVED** [review]

**Pull request merged**